README.md

```diff
@@ -168,7 +180,9 @@ end
 ```
 
 ### Attributes:
-* ```directory```: The directory on Rackspace Cloud Files where the file can be found or should be uploaded to.
+* ```filename```: The local filepath to download the file to. This is the named_attribute. 
+* ```directory```: The container on Rackspace Cloud Files where the file can be found or should be uploaded to.
+* ```filepath```:  The path on Rackspace Cloud Files where the file can be found or uploaded to. This is the path under the container.  Eg. if the full path is my_container/my_subdirectory/my_file then filepath should be my_subdirectory/my_file.  If filepath is blank, the file is assumed to be in the root of the container, with the filename taken from the filename attribute.
```

Comment:
It feels a little weird to say "use basename of filename, or filepath" -- would it be better to make it just a path to a directory/container, and still append filename?

E.g. `"#{filepath}/#{filename}"`

That way you don't have to repeat the filename twice? (Once in the resource name or filename and once again in the filepath?) Or is that flexibility good to have when you don't want the file to be named the same locally and remotely?